models/tesolin2023hint/meadata.yaml

```diff
@@ -0,0 +1,31 @@
+title: Heterogeneous wIreless Network onTology
+acronym: hint
+issued: 2023
+modified:
+contributor:
+ - https://dblp.org/pid/166/2068
+ - https://dblp.org/pid/342/7506
+ - https://dblp.org/pid/125/7740
+ - https://dblp.org/pid/c/MariaClaudiaReisCavalcanti
+keyword:
+ - mobile networks
+ - seamless handover
+ - telecommunications
+theme: Class T - Technology
+editorialNote: This model contains a n-ary relation. These relations are not supported by the current JSON exportation feature. Hence, to be able to generate the ontology.json file, this n-ary relation was temporarily excluded (but it is kept in the vpp file).
+ontologyType:
+ - domain
+language: en
+designedForTask:
+ - conceptual clarification
+ - decision support system
+ - ontological analysis
+context:
+ - research
+source:
+ - https://doi.org/10.1016/j.future.2023.08.008
```

Comment:
Should not we link both publications in the bib file here?

Reply:
Yes, it was my fault. The second identifier was added.

Reply:
ontology.ttl file added